.github/workflows/build.yml

```diff
@@ -58,6 +58,11 @@ jobs:
         with:
           usesh: true
           prepare: |
+            mkdir -p /usr/local/etc/pkg/repos
+            sed 's/quarterly/latest/' /etc/pkg/FreeBSD.conf \
```

Comment:
Can be simplified for CI as changes will be discarded on shutdown.

See also https://github.com/Alexays/Waybar/blob/710f89599e87/.github/workflows/freebsd.yml#L20

Reply:
Thank you for your review and advice. We will change to FreeBSD `latest` ports branch for the CI as well then.